### PR TITLE
httping: enable ncurses+openssl+fftw, update to 2.4

### PR DESCRIPTION
**Pull request by np**

### CODE REVIEW

File: pkgs/tools/networking/httping/default.nix [hunk 1]
```diff
@@ -1,16 +1,23 @@
...
-    url = "http://www.vanheusden.com/httping/httping-2.3.4.tgz";
-    sha256 = "1hkbhdxb0phrvrddx9kcfpqlzm41xv9jvy82nfkqa7bb0v5p2qd7";
+    url = "http://www.vanheusden.com/httping/httping-2.4.tgz";
+    sha256 = "dab59f02b08bfbbc978c005bb16d2db6fe21e1fc841fde96af3d497ddfc82084";
   };
 
-  buildInputs = [ gettext ];
+  buildInputs = [ gettext openssl ncurses fftw ];
+
+  configureFlags = [
```
Comment (np) [inline pkgs/tools/networking/httping/default.nix]: Any pointer on the documentation on how to do so?

On 03/23/2015 11:18 AM, Arseniy Seroka wrote:
> In pkgs/tools/networking/httping/default.nix
> <https://github.com/NixOS/nixpkgs/pull/6923#discussion_r26925295>:
>
> >    };
> >
> > -  buildInputs = [ gettext ];
> > +  buildInputs = [ gettext openssl ncurses fftw ];
> > +
> > +  configureFlags = [
>
> Maybe make it optional?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/NixOS/nixpkgs/pull/6923/files#r26925295>.
>

-- 
Best regards,
-- NP

Comment (jagajaga) [inline pkgs/tools/networking/httping/default.nix]: Take a look at 
https://github.com/NixOS/nixpkgs/blob/master/pkgs/top-level/all-packages.nix#L13381
and
https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/science/math/scilab/default.nix

Comment (jagajaga) [inline pkgs/tools/networking/httping/default.nix]: Or better like that https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/networking/znc/default.nix#L23

**Comment by jagajaga** [comment]
Updates?

**Comment by nbp** [comment]
Hi @np , are you still interested in fixing this issue as suggested by @jagajaga ?

**Pull request closed without merging**